src/main.rs

```diff
@@ -328,6 +432,13 @@ async fn main() -> anyhow::Result<()> {
         enable_structured_logging: args.enable_structured_logging,
         otel_config,
         policy_service_config,
+        enable_metrics: args.enable_metrics,
+        metric_backend: backend(&BackendConfig {
+            backend_type: args.metrics_type,
+            otel_metrics_endpoint: args.otel_metrics_endpoint,
+            otel_mode: args.otel_metrics_mode,
+            prometheus_port: args.metrics_prometheus_port,
+        }),
```

Comment:
How do you feel about building the metrics backend a bit more explicitly above this huge list? feels like it's somewhat buried down here.